packages/frontpage/lib/schema.ts

```diff
@@ -154,3 +160,43 @@ export const OauthSession = sqliteTable("oauth_sessions", {
   expiresAt: dateIsoText("expires_at").notNull(),
   createdAt: dateIsoText("created_at").notNull(),
 });
+
+export const ModerationEvent = sqliteTable("moderation_events", {
+  id: integer("id").primaryKey(),
+  subjectUri: text("subject_uri").notNull(),
+  subjectDid: text("subject_did").notNull(),
+  subjectCollection: text("subject_collection"),
+  subjectRkey: text("subject_rkey"),
+  subjectCid: text("subject_cid"),
+  createdBy: text("created_by").notNull(),
+  createdAt: dateIsoText("created_at").notNull(),
+  labelsAdded: text("labels_added"),
+  labelsRemoved: text("labels_removed"),
+  creatorReportReason: text("report_type"),
+});
+
+export const LabelledProfile = sqliteTable("labelled_profiles", {
+  id: integer("id").primaryKey(),
+  did: text("did").notNull().unique(),
+  isHidden: integer("is_hidden", { mode: "boolean" }).notNull().default(false),
```

Comment:
`created_at` and `updated_at` not needed? I feel like it should be here if only for consistency.